modules/match.js

```diff
@@ -1,10 +1,7 @@
 import invariant from 'invariant'
-import createMemoryHistory from 'history/lib/createMemoryHistory'
-import useBasename from 'history/lib/useBasename'
+import createMemoryHistory from './createMemoryHistory'
+import createTransitionManager from './createTransitionManager'
 import { createRoutes } from './RouteUtils'
-import useRoutes from './useRoutes'
-
-const createHistory = useRoutes(useBasename(createMemoryHistory))
```

Comment:
It's not a no-op because the `...options` spread below can include `basename`. It used to be more explicit, but I changed it to a spread because it made the docs more concise.